libraries/common/subscriptions/router.spec.js

```diff
@@ -198,7 +215,7 @@ describe('Router subscriptions', () => {
       }));
     });
 
-    it('should not redirect to a protector route when the user is logged in', () => {
+    it('should not redirect to a protector route when the user is logged in', async () => {
```

Comment:
protected

Reply:
same like above. it's about a redirect to the route which protects a protected route. this is called `protector`.